app/views/user_submitted/pages/_contents_form.erb

```diff
@@ -70,8 +79,10 @@
             }
           })
           .done(function(data) {
+            $(btn).parent().children('.status-icon').html('<i class="fa fa-check" aria-hidden="true"></i>');
+            $("#prompt").html(data["notice"]).css('display','flex');
             console.log(data, btn);
-            $(btn).parents('.dz-image-preview').hide();
+            // $(btn).parents('.dz-image-preview').hide();
```

Comment:
Looks like we no longer hide the upload box. Can they now update the text?

Reply:
Yes they can. They can save the text, modify and save again to overwrite.

Reply:
Most Excellent!